api/src/main/resources/workbench.yaml

```diff
@@ -111,6 +111,27 @@ paths:
           schema:
             $ref: "#/definitions/UsernameTakenResponse"
 
+  /api/v1/google-invitation:
```

Comment:
How is this code related to Google? In other words, what is the significance of "google" in this path?